CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to
 
```

Comment:
commit message
We do not want a user **is able to** enroll on several opened course runs => we do not want a user **to be able to** enroll

Reply:
for the commit message, this is a proposition fixing few

We do not want a user to be able to enroll on several opened course runs of the
same course. Currently, we only check if the user is already enrolled on enrollment creation. One
case is not actually covered. Indeed if the user enrolls in a first-course
 run and then unenrolls from it, he will be able to enroll in a second-course run
 and then re-enroll in the first one.